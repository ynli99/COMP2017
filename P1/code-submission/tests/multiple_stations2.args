Beijing Sydney "15:30:10"
