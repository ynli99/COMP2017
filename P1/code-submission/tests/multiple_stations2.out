The next train to Sydney from Beijing departs at 19:50:30
